corehq/apps/locations/tests/test_location_expressions.py

```diff
@@ -69,59 +69,71 @@ def test_location_not_found(self):
         self._check_expression(doc, None)
 
 
+def setup_module():
+    domain = "a-song-of-ice-and-fire"
+    domain_obj = create_domain(domain)
+    continent_location_type = LocationType(
+        domain=domain,
+        name="continent",
+        code="continent",
+    )
+    continent_location_type.save()
+    kingdom_location_type = LocationType(
+        domain=domain,
+        name="kingdom",
+        code="kingdom",
+        parent_type=continent_location_type,
+    )
+    kingdom_location_type.save()
+    city_location_type = LocationType(
+        domain=domain,
+        name="city",
+        code="city",
+        parent_type=kingdom_location_type,
+    )
+    city_location_type.save()
+
+    parent = SQLLocation(
+        domain=domain,
+        name="Westeros",
+        location_type=continent_location_type,
+        site_code="westeros",
+    )
+    parent.save()
+    child = SQLLocation(
+        domain=domain,
+        name="The North",
+        location_type=kingdom_location_type,
+        parent=parent,
+        site_code="the_north",
+    )
+    child.save()
+    grandchild = SQLLocation(
+        domain=domain,
+        name="Winterfell",
+        location_type=city_location_type,
+        parent=child,
+        site_code="winterfell",
+    )
+    grandchild.save()
+
+    globals()["domain_obj"] = domain_obj
+    globals()["domain"] = domain
+    globals()["parent"] = parent
+    globals()["child"] = child
+    globals()["grandchild"] = grandchild
```

Comment:
@dimagi/test-team curious what you all think of using module fixtures in this manner. The whole globals thing is messy, but not sure there is a cleaner way around it?